base/main.tf

```diff
@@ -350,3 +218,18 @@ resource "local_file" "helm_values_nginx-ingress_file" {
   content  = "${data.template_file.helm_values_nginx_ingress.rendered}"
   filename = "${path.cwd}/helm-values-nginx-ingress.yaml"
 }
+
+data "template_file" "helm_values_autoscaler" {
+  template = "${file("${path.module}/../templates/helm-values-autoscaler.tpl.yaml")}"
+
+  vars {
+    aws_region          = "${data.aws_region.current.name}"
+    cluster_name        = "${var.name}"
```

Comment:
This is properly namespaces? eg no staging and prod clusters int he same account will affect eachother?